.gitignore

```diff
@@ -45,3 +45,4 @@ examples/.terraform.lock.hcl
 main.tf
 workspace/*.tf*
 workspace/.terraform.d/
+workspace/tophatting.md
```

Comment:
Interesting! Do you have any fancy tools that you're using for tophatting?
🎩 
🧐 👌